task.sql

```diff
@@ -1,33 +1,55 @@
--- Create database and tables
-
 CREATE DATABASE ShopDB;
 USE ShopDB;
 
 CREATE TABLE Countries (
-    ID INT,
+    ID INT AUTO_INCREMENT,
     Name VARCHAR(50),
     PRIMARY KEY (ID)
 );
 
-CREATE TABLE ProductInventory (
-    ID INT,
-    ProductName VARCHAR(50),
-    WarehouseAmount INT,
-    WarehouseName VARCHAR(50),
-    WarehouseAddress VARCHAR(50), 
+CREATE TABLE Products (
+	ID INT AUTO_INCREMENT,
+    Name VARCHAR(50),
+    PRIMARY KEY (ID)
+);
+
+CREATE TABLE Warehouses (
+	ID INT AUTO_INCREMENT,
+    Name VARCHAR(50),
+    Address VARCHAR(50),
     CountryID INT,
 	FOREIGN KEY (CountryID) REFERENCES Countries(ID) ON DELETE NO ACTION,
     PRIMARY KEY (ID)
 );
 
--- Populate test data
+CREATE TABLE ProductInventory (
+    ID INT,
+    ProductID INT,
+    FOREIGN KEY (ProductID) REFERENCES Products(ID) ON DELETE NO ACTION,
+    Amount INT,
+    WarehouseID INT,
+    FOREIGN KEY (WarehouseID) REFERENCES Warehouses(ID) ON DELETE NO ACTION,
+    PRIMARY KEY (ID)
+);
+
 
 INSERT INTO Countries (ID,Name)
 	VALUES (1, 'Country1');
+
 INSERT INTO Countries (ID,Name)
 	VALUES (2, 'Country2');
-    
-INSERT INTO ProductInventory (ID,ProductName,WarehouseAmount,WarehouseName,WarehouseAddress,CountryID)
-	VALUES (1, 'AwersomeProduct', 2, 'Warehouse-1', 'City-1, Street-1',1);
-INSERT INTO ProductInventory (ID,ProductName,WarehouseAmount,WarehouseName,WarehouseAddress,CountryID)
-	VALUES (2, 'AwersomeProduct', 5, 'Warehouse-2', 'City-2, Street-2',2);
+
+INSERT INTO Products (Name)
+	VALUES ('AwersomeProduct');
+
+INSERT INTO Warehouses (Name, Address, CountryID)
+	VALUES ('Warehouse-1', 'City-1, Street-1', 1);
+
+INSERT INTO Warehouses (Name, Address, CountryID)
+	VALUES ('Warehouse-2', 'City-2, Street-2', 2);
+
+INSERT INTO ProductInventory (ID,ProductID,Amount,WarehouseID)
+	VALUES (1, 1, 2, 1);
+
+INSERT INTO ProductInventory (ID,ProductID,Amount,WarehouseID)
+	VALUES (2, 1, 5, 2);
```

Comment:
Add Eof(End-of-file), need to add blank line at the end
![image](https://github.com/user-attachments/assets/1700da48-df30-4775-a6a2-2443c992c0f4)